app/grandchallenge/reader_studies/forms.py

```diff
@@ -310,6 +337,52 @@ def interface_choices(self):
     def initial_interface(self):
         return self.interface_choices().first()
 
+    def widget_choices(self):
+        answer_type = self["answer_type"].value()
+        choices = [("", "Default")]
+        if answer_type:
+            try:
+                extra_options = ANSWER_TYPE_TO_QUESTION_WIDGET[answer_type]
+                choices.extend(
+                    (option.name, option.label) for option in extra_options
+                )
+            except KeyError:
+                pass
+        return choices
+
+    def initial_widget(self):
+        try:
+            return self.instance.widget.kind
+        except ObjectDoesNotExist:
+            return ""
```

Comment:
Does this return value need to match the default choice? Same in L361 (`if value == ""`). If so, using `BLANK_CHOICE_DASH` would help readability.

Reply:
Yes it does, but `BLANK_CHOICE_DASH` returns `[('', '---------')]` and you can't just get the `''`. I could alternatively add the default choice to the `QuestionWidgetKindChoices` choices and then use that in all places for consistency and clarity?

Reply:
`BLANK_CHOICE_DASH[0][0]`